modules/standard/IO.chpl

```diff
@@ -9078,7 +9081,7 @@ proc fileWriter.writeHelper(const args ...?k, style:iostyleInternal) throws {
 // documented in varargs version
 @chpldoc.nodoc
 proc fileWriter.writeln() throws {
-  try this.write(new ioNewline());
+  try this.write(new chpl_ioNewline());
```

Comment:
Ideally we will eventually be writing these without the ``chpl_ioNewline`` crutch, but I think this is fine for now.